lint-requirements.txt

```diff
@@ -1,20 +1,24 @@
-#
-# This file is autogenerated by pip-compile with Python 3.12
-# by the following command:
-#
-#    pip-compile lint-requirements.in
-#
-black==24.8.0
+# This file was autogenerated by uv via the following command:
+#    uv pip compile lint-requirements.in
```

Comment:
I recommend `--universal`

Reply:
Thanks, I did that but I totally expect this to be overwritten by dependabot anyway.